build.py

```diff
@@ -124,9 +124,30 @@ def run(config, start_class_path, arguments):
 ### def run ###
 
 
+# USAGE
+#
+# Print basic usage info.
+#
+def usage(args) :
+  print('Unknown command "%s"' % args)
```

Comment:
Python will convert the list to a string. So I guess the question is, do we want
`Unknown command unsupported-command` or `Unknown command [ 'unsupported-command', 'command-arg-0', command-arg-1']`?